lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -194,6 +191,10 @@ def applyConfig(self, configdict, depth=-1):
                 'widgetClassName'), modulename=configdict.get('widgetModuleName', None))
             if isinstance(self.widget(), BaseConfigurableClass):
                 self.widget().applyConfig(configdict['widget'])
+            if hasattr(w, "setCustomWidgetMap") and hasattr(w, "getCustomWidgetMap"):
+                tmp = self._mainwindow.getCustomWidgetMap()
```

Comment:
a very minor/unimportant comment: it would be better to use a more descriptive name for the `tmp`  variable (maybe `widget_map`